src/aws/ec2-wrapper.ts

```diff
@@ -80,22 +64,22 @@ export class EC2Wrapper {
     );
 
     if (subnets.length === 0) {
-      throw new Error("Invalid subnet name, it does not exist");
+      throw new Error(`Subnets with vpc id '${vpcId}', tag key '${tagKey}' and tag values '${tagValues}' do not exist`);
     }
 
-    const missingSubnetNames = subnetNames.filter((subnetName) => {
+    const missingSubnetValues = tagValues.filter((tagValue) => {
```

Comment:
This could be a better name to make it clear that this var is a boolean
```suggestion
    const hasMissingSubnets = tagValues.filter((tagValue) => {
```

Reply:
`missingSubnetValues` is an array of missing values